choice.rb

```diff
@@ -0,0 +1,82 @@
+require_relative './albumaction/albumact'
+require_relative './books/book_ui'
+require_relative './game/game_ui'
+
+class Choice
+  OPTIONS = {
+    1 => :list_books,
+    2 => :list_music_albums,
+    3 => :list_games,
+    4 => :list_genres,
+    5 => :list_labels,
+    6 => :list_authors,
+    7 => :add_book,
+    8 => :add_music_album,
+    9 => :add_game,
+    0 => :exit
+  }.freeze
+
+  def initialize(book_app, game)
+    @book_app = book_app
+    @game = game
+  end
+
+  def run
+    input = gets.chomp.to_i
+    if OPTIONS[input]
+      send(OPTIONS[input])
+    else
+      puts 'Invalid input'
+    end
+  end
+
+  def list_books
+    puts '1 - List all books'
```

Comment:
- [ ] The "list all books" and "add a book" methods actually not calling any methods. Please fix this issue.
![image](https://github.com/ZewdieMc/catalog-of-my-things/assets/91486791/cb02f802-f725-4f37-8353-d95dc3b0b06a)
![image](https://github.com/ZewdieMc/catalog-of-my-things/assets/91486791/ac76ca2d-770a-4ec4-8869-28b62aedb4fd)